patches/src/main/resources/youtube/settings/host/values/strings.xml

```diff
@@ -1565,12 +1569,12 @@ High quality may be unlocked on some videos that require high device dimensions,
 • Video playback will use more internet data than VP9.
 • VP9 codec is still used for HDR video."</string>
     <string name="revanced_disable_vp9_codec_summary_off">VP9 codec is enabled.</string>
-    <string name="revanced_replace_av1_codec_title">Replace software AV1 codec</string>
-    <string name="revanced_replace_av1_codec_summary">Replaces the software AV1 codec with the VP9 codec.</string>
     <string name="revanced_reject_av1_codec_title">Reject software AV1 codec response</string>
     <string name="revanced_reject_av1_codec_summary">"Forcefully rejects the software AV1 codec response.
-A different codec will be applied after about 20 seconds of buffering."</string>
+        A different codec will be applied after about 20 seconds of buffering."</string>
     <string name="revanced_reject_av1_codec_toast">Fallback process causes about 20 seconds of buffering.</string>
+    <string name="revanced_replace_av1_codec_title">Replace software AV1 codec</string>
```

Comment:
Since the `Replace software AV1 codec` setting is more recommended, it is placed before `Reject software AV1 codec response`, even though it is not suitable for lexicographic arrangement.